docs/source/initial.rst

```diff
@@ -0,0 +1,73 @@
+.. _initial:
+
+#############
+Initial Setup
+#############
+
+.. important:: THIS IS THE COZMO SDK BETA. The SDK is under development and is subject to change.
+
+To use the Cozmo SDK, the Cozmo mobile app must be installed on your mobile device and that device must be tethered to a computer via USB cable.
+
+-------------
+Prerequisites
+-------------
+
+* Python 3.5.1 or later
+* WiFi connection
+* An iOS or Android mobile device with the Cozmo app installed, connected to the computer via USB cable
+
+--------------------
+SDK Example Programs
+--------------------
+
+Anki provides example programs for novice and advanced users to run with the SDK. Download the SDK example programs here:
+
+  * `macOS/Linux SDK Examples <http://cozmosdk.anki.com/0.7.0/cozmo_sdk_examples_0.7.0.tar.gz>`_
+
+  * `Windows SDK Examples <http://cozmosdk.anki.com/0.7.0/cozmo_sdk_examples_0.7.0.zip>`_
+
+Once downloaded, extract the packaged files to a new directory.
+
+------------
+Installation
+------------
+
+To install the SDK on your system, select the instructions for your computer's operating system.
+
+* :ref:`install-osx`
```

Comment:
This link has changed and is now unfortunately broken. It should be changed to install-macos.